### PR TITLE
Fix kubeadm init kernel validator display message error

### LINKED ISSUE #1051: kubeadm init validating kernel version display error
## Versions

**kubeadm version** (use `kubeadm version`): v1.11.2

**Environment**:
- **Kubernetes version** (use `kubectl version`): v1.11.2
- **Cloud provider or hardware configuration**:
- **OS** (e.g. from /etc/os-release): CentOS Linux release 7.2.1511 (Core)
- **Kernel** (e.g. `uname -a`):  3.10.0-327.el7.x86_64
- **Others**:

## What happened?
execute "kubeadm init --kubernetes-version=v1.11.2 --apiserver-advertise-address 192.168.8.64 --pod-network-cidr=10.244.0.0/16" appear kernel version display error.

The display message:
[init] using Kubernetes version: v1.11.2
[preflight] running pre-flight checks
I0813 01:14:34.397832   32483 kernel_validator.go:81] Validating kernel version
I0813 01:14:34.398067   32483 kernel_validator.go:96] Validating kernel config
[preflight/images] Pulling images required for setting up a Kubernetes cluster
[preflight/images] This might take a minute or two, depending on the speed of your internet connection
[preflight/images] You can also perform this action in beforehand using 'kubeadm config images pull'

I think Validating kernel version message display as same with "[preflight] running pre-flight checks",
change validating kernel message  `I0813 01:14:34.397832   32483 kernel_validator.go:81] Validating kernel version` to `[preflight] Validating kernel version` , change `I0813 01:14:34.398067   32483 kernel_validator.go:96] Validating kernel config` to `[preflight] Validating kernel config`

### PR DESCRIPTION
**Pull request by Farice4**
Signed-off-by: Yuanbin.Chen <cybing4@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes kubernetes/kubeadm#1051

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by kad** [comment]
I'm not sure that this patch is correct. Most of the preflight checks (`cmd/kubeadm/preflight/checks.go`) have consistent output via
```go
glog.V(1).Infoln("validating....")
```

File: cmd/kubeadm/app/util/system/kernel_validator.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func (k *KernelValidator) Validate(spec SysSpec) (error, error) {
 
 // validateKernelVersion validates the kernel version.
 func (k *KernelValidator) validateKernelVersion(kSpec KernelSpec) error {
-	glog.Infof("Validating kernel version")
+	fmt.Println("[preflight] Validating kernel version")
```
Comment (neolit123) [inline cmd/kubeadm/app/util/system/kernel_validator.go]: we shouldn't imply pre-flight checks in the validator code.

we should do this instead:
`glog.V(1).Infof(...`

for both.

File: cmd/kubeadm/app/util/system/kernel_validator.go [hunk 2]
```diff
@@ -78,7 +78,7 @@ func (k *KernelValidator) Validate(spec SysSpec) (error, error) {
 
 // validateKernelVersion validates the kernel version.
 func (k *KernelValidator) validateKernelVersion(kSpec KernelSpec) error {
-	glog.Infof("Validating kernel version")
+	glog.V(1).Infof("Validating kernel version")
```
Comment (bart0sh) [inline cmd/kubeadm/app/util/system/kernel_validator.go]: Why to use Infof here? We're not formatting anything, so just glog.V(1).Info should be ok.

**Comment by kad** [comment]
@chenyb4 please also fix `Infof` -> `Info`.

**Review by timothysc — APPROVED** [review]
/approve

**Pull request merged**